draft-ietf-taps-interface.md

```diff
@@ -1284,7 +1284,9 @@ application Requires the use of temporary addresses, the resulting Connection
 cannot use IPv4, because temporary addresses do not exist in IPv4. Second,
 temporary local addresses might involve trading off privacy for performance.
 For instance, temporary addresses can interfere with resumption mechanisms
-that some protocols rely on to reduce initial latency.
+that some protocols rely on to reduce initial latency. See
+{?I-D.gont-v6ops-ipv6-addressing-considerations} for more discussion on the
```

Comment:
I'm a bit concerned about adding a reference to an individual draft here. Could we find a better reference for temporary addresses?

Reply:
IT could BE ENOUGH to just add a ref: "For instance, temporary addresses (e.g. RFC8981) that can interfere with resumption mechanisms?

Reply:
Yes, I'd prefer that ref.